controllers/testutil/test_util.go

```diff
@@ -34,7 +34,7 @@ import (
 
 // CreateNSName generates random namespace names. Namespaces are never deleted in test environment
 func CreateNSName(prefix string) string {
-	suffix := make([]byte, 20)
+	suffix := make([]byte, 10)
```

Comment:
I am fine with this change, but any reason why we do that?

Reply:
For the annotation `key/value`, the `key` should be less than 63 characters.
So if we use 20 bytes suffix, it will convert to 40 characters because of %x, base 16, two characters per byte.
`key.Namespace+"."+key.Name` will be longer than 63.

Reply:
Oh, good point. Nice to know that.